<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>420</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame_login">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>351</width>
       <height>469</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_login_accountNumber">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_pass">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login_accountNumber">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login_pass">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_forgetPass">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_img">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_login_img">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QFrame" name="frame_forgetPass">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>560</width>
       <height>420</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_forget_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>560</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_forgetMsg">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>331</width>
        <height>291</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_forget_accountNumber">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_forget_accountNumber">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_forget_pass1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_forget_pass1">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_forget_pass2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_forget_pass2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_getCode">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>170</y>
         <width>55</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_forget_code">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_forget_commit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>110</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_forget_cancel">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>110</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_forget_img">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
